primary.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_instance" "primary" {
-  count         = "${var.primary_count}"
+  count         = "${var.install_type == "ipm" ? 3 : var.primary_count}"
```

Comment:
Can we add a comment along side this resource with the why? With a hard coded value we could very well come back and not recall the context for this.